src/Simpleue/Locker/LockerInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Simpleue\Locker;
+
+interface LockerInterface
```

Comment:
Suffix "Interface" should be removed. It is not used in other interfaces as "Job" or "Queue".